.github/workflows/auto-delete-on-schedule.yml

```diff
@@ -56,26 +77,6 @@ jobs:
   delete-check:
     needs: [rosa-delete, aurora-delete, route53-delete]
```

Comment:
Wouldn't adding `accelerator-delete` fix the issue?

Reply:
I don't think so. The setup we currently have is the following: `[kc1,kc2,kc3] <- LB-site-1 <- GA -> LB-site2 -> [kc1,kc2,kc3]`. Removal of GA is failing when LBs are not available. My assumption is that the LBs are removed as part of the keycloak deployment removal/ROSA clusters removal. Therefore we need to make sure GA removal is done before ROSA clusters are removed.

Reply:
gotcha. How about Route53? Does it have the same problem?

Reply:
No because we are not using tofu for removing it. Since this: https://github.com/keycloak/keycloak-benchmark/blob/main/provision/opentofu/modules/aws/accelerator/main.tf#L37 is an input for creating GA it needs to be available also when destroying the state even though it should not be necessary for removing it. Maybe we should change this to have more flexible way for removing GAs. Let's see how this fix works and if we will have some unremoved GAs in future we can rethink how we do it.